assets/src/edit-story/components/dropDown/index.js

```diff
@@ -0,0 +1,320 @@
+import { debounce } from 'throttle-debounce';
+
+/**
+ * WordPress dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as DropDownIcon } from '../../icons/dropdown.svg';
+
+const DropDownContainer = styled.div`
+  width: 100px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  color: ${({ theme }) => theme.colors.fg.v0};
+  font-family: ${({ theme }) => theme.fonts.body1.font};
+`;
+
+const DropDownSelect = styled.div.attrs({ role: 'button', tabIndex: '0' })`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  flex-grow: 1;
+  background-color: ${({ theme }) => rgba(theme.colors.bg.v0, 0.3)};
+  border-radius: 4px;
+  padding: 2px 0 2px 6px;
+  cursor: pointer;
+
+  ${({ disabled }) =>
+    disabled &&
+    `
+      pointer-events: none;
+      opacity: 0.3;
+    `}
+
+  svg {
+    width: 28px;
+    height: 28px;
+    color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.3)};
+  }
+`;
+
+const DropDownTitle = styled.span`
+  user-select: none;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+`;
+
+const DropDownListWrapper = styled.div``;
+
+const DropDownList = styled.ul`
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  float: left;
+  min-width: 160px;
+  width: 100%;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  font-size: 14px;
+  text-align: left;
+  list-style: none;
+  background-color: #fff;
+  background-clip: padding-box;
+  border: 1px solid #ccc;
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+
+  li {
+    cursor: pointer;
+    :hover {
+      background: #ddd;
+    }
+    & > a {
+      display: block;
+      padding: 5px 20px;
+      clear: both;
+      font-weight: 400;
+      line-height: 1.42857143;
+      color: #333;
+      white-space: nowrap;
+      text-decoration: none;
+    }
+  }
+`;
+
+const DropDownItem = styled.li.attrs({ tabIndex: '0', role: 'option' })`
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+  padding: 16px;
+  margin: 0;
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+  }
+
+  &:focus {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+    outline: none;
+  }
+`;
+
+const DropDownItemSpan = styled.a``;
```

Comment:
Why use `<a />` here? Also, since it doesn't have style, what about using the element directly in the component instead to create a styled one?

Reply:
Good point. Removed